src/assessor_tools.yml

```diff
@@ -86,13 +100,13 @@
           - --strip-components=1
     - role: assessment_tool
       vars:
-        archive_src: "https://github.com/ChrisTruncer/Egress-Assess/\
-        tarball/master"
+        # TODO: Switch to master branch after Forked branch is merged
+        # See https://github.com/cisagov/kali-packer/issues/139
```

Comment:
Merging the branches was a recommendation and was originally the intent for the tool when it was created. There were some issues encountered when originally modifying the tool that led to the creation of the Forked branch to further testing. Ideally it will be merged but no definitive plans have been done to do so but I will be following up with the developer to request that it happen. So for now this is here as a reminder just in case it happens and we need to go back and update the branch.

Reply:
I agree with @mcdonnnj that here should really be an issue in [tarrell13/Egress-Assess](https://github.com/tarrell13/Egress-Assess) to track:
- Why the `Forked` branch was created
- Why the `Forked` branch cannot be merged into `master` right now

Include references to #139 and cisagov/egress-assess-packer#13.  You should also note that the further `master` and `Forked` grow apart the less likely it is that you will be able to perform a clean merge.

Reply:
Understood. Issues are currently not enabled on the forked version of the tool. I have reached out to creator to enable issues on the fork and will let them know that the longer it takes to merge the forks the messier it will be.

Reply:
Intentionally disabling issues for a repo is pretty silly, so the developer should have no problem enabling them.

Given that @tarrell13 (the developer) works for CISA, it wouldn't be unreasonable to transfer his fork of [`FortyNorthSecurity/Egress-Assess`](https://github.com/FortyNorthSecurity/Egress-Assess) to the `cisagov` organization.  Please let us know if he wants to do that.

Reply:
For now we're going to stick with the Forked branch. Please let me know what else is needed to resolve this so we can get these changes implemented. The hope was to have this ready for the next env builds scheduled to begin next week.

Reply:
@mcdonnnj In the interest of getting this new AMI rolled out ASAP, can you please revisit this conversation and resolve if you're satisfied?

@m1j09830 Please let us know when an issue has been created in [tarrell13/Egress-Assess](https://github.com/tarrell13/Egress-Assess).

Reply:
@tarrell13 ended up making a completely new [repository](https://github.com/tarrell13/Auto-Egress-Assess) to mitigate the issues with trying to merge the branches. We will be using this new repository moving forward and the changes have been incorporated into the `assessor-tools.yml` file in https://github.com/cisagov/kali-packer/commit/e06b3432e21f269d380abf6225c3a30d645ca2d2.